README.md

```diff
@@ -272,3 +272,23 @@ add_subdirectory("path/to/source/of/openPMD-api")
 
 target_link_libraries(YourTarget PRIVATE openPMD::openPMD)
 ```
+
+### Manually
+
+If your (Linux/OSX) project is build by calling the compiler directly or uses a manually written `Makefile`, consider using our `openPMD.pc` helper file for `pkg-config` which are installed alongside the library.
+
+First set the following environment hint if openPMD-api was *not* installed in a system path:
+
+```bash
+# optional: only needed if installed outside of system paths
+export PKG_CONFIG_PATH=$HOME/somepath/lib/pkgconfig:$PKG_CONFIG_PATH
+```
+
+Additional linker and compiler flags for your project are available via:
+```bash
+pkg-config --libs openPMD
+# -L${HOME}/somepath/lib -lopenPMD
```

Comment:
note: the `--static` output requires proper listing of `Libs.private` which is non-trivial. Luckily, we changed the default build variant to `shared` #505 for the next release, so `pkg-config` installs are only supported with shared builds for now, which is a sane limitation.